### PR TITLE
Add link to robot documentation from robot page

### PR DESCRIPTION
**Pull request by Eddasol**

### CODE REVIEW

**Comment by Eddasol** [comment]
Dependent on https://github.com/equinor/isar/pull/604

File: frontend/src/components/Pages/RobotPage/Documentation.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    align-items: center;
+`
+
+export const DocumentationSection = ({ documentation }: { documentation: DocumentInfo[] }) => {
+    const { TranslateText } = useLanguageContext()
+
+    return (
+        <>
+            <Typography variant="h2">{TranslateText('Documentation')}</Typography>
+            {documentation.map((documentInfo, index) => DocumentDisplay(index, documentInfo))}
```
Comment (andchiind) [inline frontend/src/components/Pages/RobotPage/Documentation.tsx:20]:
```suggestion
            {documentation.map((documentInfo, index) =><DocumentDisplay key={index} documentInfo={documentInfo} />)}
```

Comment (andchiind) [reply frontend/src/components/Pages/RobotPage/Documentation.tsx:20]: Though in this case you might need to rename "key"

**Review by andchiind — APPROVED** [review]
LGTM

**Pull request merged**